docs/types.rst

```diff
@@ -96,7 +96,19 @@ In Solidity, division rounds towards zero. This mean that ``int256(-5) / int256(
 Note that in contrast, division on :ref:`literals<rational_literals>` results in fractional values
 of arbitrary precision.
 
-Division by zero and modulus with zero throws a runtime exception.
+The modulo operator ``%`` yields the remainder ``r`` after the division of the left operand ``a``
```

Comment:
Perhaps start with ``The result of the modulo operation `a % n`...``, then you do not have to explain `a` and `n` anymore.